management/azure_mgmt_analysis_services/lib/profiles/Profile_2016_05_16/modules/analysisservices_profile_module.rb

```diff
@@ -0,0 +1,94 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+require_relative '../../../azure_mgmt_analysis_services'
+
+
+module Azure::Profiles::AnalysisServicesModule::Management::Profile_2016_05_16
+  module AnalysisServices
+    Servers = Azure::ARM::AnalysisServices::Api_2016_05_16::Servers
+
+    module Models
+      ServerAdministrators = Azure::ARM::AnalysisServices::Api_2016_05_16::Models::ServerAdministrators
+      AnalysisServicesServerUpdateParameters = Azure::ARM::AnalysisServices::Api_2016_05_16::Models::AnalysisServicesServerUpdateParameters
+      Resource = Azure::ARM::AnalysisServices::Api_2016_05_16::Models::Resource
+      SkuEnumerationForNewResourceResult = Azure::ARM::AnalysisServices::Api_2016_05_16::Models::SkuEnumerationForNewResourceResult
+      AnalysisServicesServers = Azure::ARM::AnalysisServices::Api_2016_05_16::Models::AnalysisServicesServers
+      SkuDetailsForExistingResource = Azure::ARM::AnalysisServices::Api_2016_05_16::Models::SkuDetailsForExistingResource
+      ResourceSku = Azure::ARM::AnalysisServices::Api_2016_05_16::Models::ResourceSku
+      SkuEnumerationForExistingResourceResult = Azure::ARM::AnalysisServices::Api_2016_05_16::Models::SkuEnumerationForExistingResourceResult
+      AnalysisServicesServer = Azure::ARM::AnalysisServices::Api_2016_05_16::Models::AnalysisServicesServer
+      SkuTier = Azure::ARM::AnalysisServices::Api_2016_05_16::Models::SkuTier
+      State = Azure::ARM::AnalysisServices::Api_2016_05_16::Models::State
+      ProvisioningState = Azure::ARM::AnalysisServices::Api_2016_05_16::Models::ProvisioningState
+    end
+
+    #
+    # AnalysisServices
+    #
+    class AnalysisServicesClass
+      attr_accessor :servers, :configurable, :base_url, :options, :model_classes
+
+      def initialize(configurable, base_url, options)
+        @configurable, @base_url, @options = configurable, base_url, options
+        client = Azure::ARM::AnalysisServices::Api_2016_05_16::AnalysisServicesManagementClient.new(configurable.credentials, base_url, options)
+        if(client.respond_to?(:subscription_id))
+          client.subscription_id = configurable.subscription_id
+        end
+        self.servers = Azure::ARM::AnalysisServices::Api_2016_05_16::Servers.new(client)
+        self.model_classes = ModelClasses.new
+      end
+
+      def get_client(version)
+        case version
```

Comment:
Shouldn't we have default here? `version` should be optional I think

Reply:
We could have multiple version of client possible in a single rollup profile. So, we cannot leave it optional in rollup. But we could leave it optional in individual profiles. Code changes done